ucm2/Intel/sof-essx8336/sof-essx8336.conf

```diff
@@ -20,10 +20,11 @@ SectionUseCase."HiFi" {
 If.speaker {
 	Condition {
 		Type ControlExists
-		Control "DAC Playback Volume"
+		Control "name='DAC'"
 	}
 	True.BootSequence [
-		cset "name='DAC Playback Volume' 60%"
+		cset "name='DAC' 60%"
+		cset "name='DAC Source Mux' 0"
```

Comment:
@mchehab are you sure about this one?

in https://github.com/thesofproject/linux/issues/3248#issuecomment-1060133008 @yangxiaohua2009 suggested this control name?

Reply:
No. Please see the updated version. The name of the control is correct, and setting it on higher values than 60% may cause level saturation.
So, basically, what I did at the newest version is to place this setting at SectionVerb.EnableSequence at patch 1 
, and, since the same volumes are there for both headphone and speaker, I dropped those if's at patch 3, and fixed the control names on patch 4.